src/main/java/org/galatea/starter/domain/MongoDocument.java

```diff
@@ -0,0 +1,28 @@
+package org.galatea.starter.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import java.time.Instant;
+import java.time.OffsetTime;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.galatea.starter.utils.MongoDocSerializer;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@JsonSerialize(using = MongoDocSerializer.class)
+@NoArgsConstructor
+@Document
+@Data public class MongoDocument {
```

Comment:
Consider naming this something closer to the function it's serving, like StockTick or StockData, or something similar.